R/write.R

```diff
@@ -17,11 +17,9 @@
 #' @details
 #'   * Variable and dataset labels are stored in the "label" attribute.
 #'
```

Comment:
We can keep the lines for the SAS length and replace 'SASlength' by 'width' :
#' * SAS length are stored in the "width" attribute.